src/module/scene/helpers.ts

```diff
@@ -96,6 +96,11 @@ function computeSightAndDetectionForRBV(token: TokenDocumentPF2e | PrototypeToke
         const range = scene?.flags.pf2e.hearingRange ?? null;
         token.detectionModes.push({ id: "hearing", enabled: true, range });
     }
+
+    const thsense = actor.perception.senses.get("thoughtsense");
+    if (thsense) {
+        token.detectionModes.push({ id: "senseThoughts", enabled: true, range: thsense.range });
+    }
```

Comment:
```suggestion
    const thoughtsense = actor.perception.senses.get("thoughtsense");
    if (thoughtsense) {
        token.detectionModes.push({ id: "senseThoughts", enabled: true, range: thoughtsense.range });
    }
```
This shouldn't be added if it's of vague acuity.

Reply:
Test for `thoughtsense.acuity !== "vague"` added